devel/libenkf/src/hook_manager.c

```diff
@@ -36,10 +36,8 @@
 
 
 struct hook_manager_struct {
-  char                   * path;
-  workflow_type          * workflow;
-  ert_workflow_list_type * workflow_list;
-  runpath_list_type      * runpath_list;
+  hook_workflow_type     * hook_workflow;
+  hook_workflow_type     * post_hook_workflow;
```

Comment:
I think these should be container types - probably `vector_type`. Maybe also a good idea to have a separate container manage the storage of the workflows, then the other lists can have the same reference multiple times.